app/routes/lajidata.tsx

```diff
@@ -0,0 +1,58 @@
+import { json, LoaderFunction } from "@remix-run/node";
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  const bounds = url.searchParams.get("bounds");
+  const accessToken = process.env.LAJI_ACCESS_TOKEN;
+
+  if (!bounds || !accessToken) {
+    console.error("Missing bounds or access token.");
+    return json(
+      { data: [], message: "Missing required bounds or access token" },
+      { status: 400 },
+    );
+  }
+  if (!accessToken) {
```

Comment:
You could also just throw an error. The access token missing shouldn't be an usual scenario to happen.

Reply:
I separated the two now correctly but didnt add a throw error as it is not an usual scenario to happen.

Reply:
That's fine. 👍🏻